src/mgos_provision_btn.c

```diff
@@ -15,11 +15,11 @@
  * limitations under the License.
  */
 
-#include "mgos_provision.h"
-
 #include "mgos.h"
 #include "mgos_event.h"
 #include "mgos_gpio.h"
+#include "mgos_provision.h"
```

Comment:
looks like you're using a modern version of clang-format that reorders includes by default.
mgos_provision.h, though, is meant to be first (to ensure it can stand on its own and does not transitively depend on anything), so please undo this part.
i also add `IncludeBlocks: Preserve` to .clang-format to prevent this from happening in the future.

Reply:
and rebase on latest master, please